examples/js/plugins/FeatureToolTip.js

```diff
@@ -84,17 +84,21 @@ var FeatureToolTip = (function _() {
             style = (geometry.properties && geometry.properties.style) || feature.style || layer.style;
             var context = { globals: {}, properties: getGeometryProperties(geometry) };
             style = style.drawingStylefromContext(context);
-            fill = style.fill.color;
-            stroke = '1.25px ' + style.stroke.color;
 
             if (feature.type === itowns.FEATURE_TYPES.POLYGON) {
                 symb = '&#9724';
+                fill = style.fill.color;
```

Comment:
could you add check if `style` and `style.fill` exits?

Reply:
I don't know if this is necessary in curent itowns version.
If I'm not mistaking, `FeatureToolTip` doesn't work on polygons displayed within a `GeometryLayer`. It thus only work in a `ColorLayer`.
If you display some polygons within a `ColorLayer` without specifying a `fill` property for the `Style` of the layer, the polygons won't be displayed. Therefore, I believe that there is no case where you try to display a tool tip on a polygon and the `Style.fill` property is undefined.
Should I add a check for `style` and `style.fill` anyways ?

Reply:
The polygon isn't displayed but polygon exists in `FeatureCollection`?
Could you try, in example, if you remove `fill.color` in style?

Reply:
Actually, after checking acording to your latest indications, you are right : 
- On the [kml example](http://www.itowns-project.org/itowns/examples/index.html#source_file_kml_raster), the polygons appear even if they is no definition for `fill`.
- On the [geojson example](http://www.itowns-project.org/itowns/examples/index.html#source_file_geojson_raster), the polygons don't appear when there is no `fill`, `stroke`, or even no `Style` definition in the layer. Yet, they exists in `FeatureCollection`. Therefore a tooltip is displayed when hovering their position with cursor (event if the polygons are not visible).

I shall add the checks for `style`, `style.fill` and `style.stroke`

Reply:
Furthermore, if you add a polygon with no `fill` but a `stroke`, the tool tip cannot display the legend correctly if `style.fill` is not tested, hence the need of a check as you suggested.